src/sentry/migrations/0101_backfill_file_type_on_event_attachment.py

```diff
@@ -19,8 +19,9 @@ def backfill_file_type(apps, schema_editor):
     EventAttachment = apps.get_model("sentry", "EventAttachment")
     all_event_attachments = EventAttachment.objects.select_related("file").all()
     for event_attachment in RangeQuerySetWrapper(all_event_attachments, step=1000):
-        event_attachment.type = event_attachment.file.type
-        event_attachment.save(update_fields=["type"])
+        if event_attachment.type is None:
+            event_attachment.type = event_attachment.file.type
+            event_attachment.save(update_fields=["type"])
```

Comment:
Wait, I think I misunderstood this change - I guess we should check `event_attachment.type != event_attachment.file.type` instead

Reply:
Actually, I guess it's fine either way